src/Umbraco.Web.UI.Client/src/views/components/users/umb-user-group-preview.html

```diff
@@ -1,7 +1,7 @@
 <div class="umb-user-group-preview">
-    
-    <i ng-if="icon" class="umb-user-group-preview__icon {{ icon }}"></i>
-    
+
+    <umb-icon icon="{{icon}}" class="umb-user-group-preview__icon"></umb-icon>
```

Comment:
@nul800sebastiaan it also need `{{icon}}` in class attribute for now in case a (custom) font icon is used.

Reply:
👍 Thanks, amended! https://github.com/umbraco/Umbraco-CMS/commit/ef3495b16a10fda695ce1cd541b1f07bed284943